lib/search_cop.rb

```diff
@@ -44,26 +44,26 @@ def search_scope(name, &block)
       search_scopes[name] = SearchScope.new(name, self)
       search_scopes[name].instance_exec(&block)
 
-      send(:define_singleton_method, name) { |query, query_options = {}| search_cop(query, name, query_options) }
-      send(:define_singleton_method, "unsafe_#{name}") { |query, query_options = {}| unsafe_search_cop(query, name, query_options) }
+      send(:define_singleton_method, name) { |query, query_options = {}, scope_options = {}| search_cop(query, name, query_options, scope_options) }
```

Comment:
The syntax would be cleaner with keyword arguments. Is that possible to do without a breaking change?